frontend/app/types/card.ts

```diff
@@ -4,6 +4,7 @@ export interface CardData {
   meaning: string;
   explanation: string;
   examples: string[];
+  depiction?: string[];
```

Comment:
Here we should add all the new fields in the new idioms model, e.g. Category, frequency of use, the three different depictions etc